test_integrated.py

```diff
@@ -0,0 +1,44 @@
+import subprocess
+import unittest
+from time import sleep
+
+import certifi
+import pymongo
+
+import ortho
+from session import start_session
+
+
+class MyTestCase(unittest.TestCase):
+    def test_ex_nihilo(self):
```

Comment:
`MyTestCase` could use a rename. :) I'd also suggest swapping to pytest and getting rid of the class entirely. (You can also organize tests into classes later with pytest if you'd like.)